src/blueprint_provider/default_vm_blueprint_provider.cpp

```diff
@@ -110,6 +110,11 @@ mp::Query mp::DefaultVMBlueprintProvider::fetch_blueprint_for(const std::string&
     Query query{"", "default", false, "", Query::Type::Alias};
     auto& blueprint_config = blueprint_map.at(blueprint_name);
 
+    if (!blueprint_config["instances"][blueprint_name])
+    {
+        throw InvalidBlueprintException(fmt::format("Definition of \"{}\" not found", blueprint_name));
```

Comment:
I think having something like
```
fmt::format("There are no instance definitions matching Blueprint name \"{}\"", blueprint_name));
```
makes it a bit more clear what the error is.

Reply:
Right. Done.